packages/common/src/converter/helpers.ts

```diff
@@ -0,0 +1,24 @@
+export function patchRoot<T>(root: T, seen = new Set()): T {
+  for (const key in root) {
```

Comment:
Why not just go over the `nested` attributes?

Reply:
I didn't realize capitalized namespaces were both at root and in nested. Updated.